domains/Vakarux.json

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Vakarux's is-a.dev domain, for now it's just a redirect to my long domain that will expire soon. For now.",
+  "owner": {
+    "username": "Vakarux",
+    "email": "vakaris@amva.pro"
+  },
+  "record": {
+    "URL": "https://profectus-bonus.eu"
```

Comment:
that looks like some template, according to
- Phone number: 123456789
- Address: "Address 1"